lib/helpers.js

```diff
@@ -90,6 +91,28 @@ function getIMEListFromOutput (stdout) {
   return engines;
 }
 
+async function getSdkToolsVersion() {
```

Comment:
Appium's `fs` replacement has a `readFile` method (https://github.com/appium/appium-support/blob/master/lib/fs.js). So this becomes something like
```js
import { fs } from 'appium-support';
import path from 'path';

// ...

let sourceProperties = path.resolve(tools, 'source.properties');
let data = await fs.readFile(sourceProperties, 'utf8'));
let version;

// either
for (let line of data.split('\n')) {
  if (line.indexOf('Pkg.Revision') === 0) {
    version = line.substr(line.indexOf('='));
    break;
  }
}

// or 
let re = new RegExp('^Pkg\.Revision=([\d\.]+)$', 'gm');
let match = re.exec(data);
version = match[1];

log.debug(`Using Android version: ${version}`);
```